### PR TITLE
Fix name of Ubuntu prerequisite

### PR DESCRIPTION
**Pull request by kenstir**
Change name to gir1.2-appindicator3-0.1.  Tested on Ubuntu 22.04.4 LTS and Ubuntu 20.04.6 LTS.

### CODE REVIEW

**Review by cas-- — CHANGES_REQUESTED** [review]
We actually support the newer `AyatanaAppIndicator3` so would be better to update to [gir1.2-ayatanaappindicator3-0.1](https://packages.ubuntu.com/focal/gir1.2-ayatanaappindicator3-0.1). Also if you do a quick search I think there is another reference in the docs that could be updated

**Review by kenstir — COMMENTED** [review]
self check: pass

File: DEPENDS.md [hunk 1]
```diff
@@ -95,6 +95,6 @@ All modules will require the [common](#common) section dependencies.
 [mako]: https://www.makotemplates.org/
 [pygame]: https://www.pygame.org/
 [libnotify]: https://developer.gnome.org/libnotify/
-[python-appindicator]: https://packages.ubuntu.com/xenial/python-appindicator
+[ayatanaappindicator3]: https://packages.ubuntu.com/jammy/gir1.2-ayatanaappindicator3-0.1
```
[the remark this answers is not in the document]
Comment (cas--) [reply DEPENDS.md:98]: Actually I think this page would be better: https://lazka.github.io/pgi-docs/AyatanaAppIndicator3-0.1/index.html

[the remark this answers is not in the document]
Comment (kenstir) [reply DEPENDS.md]: That makes sense.  Done.

**Comment by cas--** [comment]
Thanks for the PR :+1:

**Pull request closed without merging**